rmgpy/molecule/draw.py

```diff
@@ -1041,12 +1046,17 @@ def _draw_line(self, cr, x1, y1, x2, y2, dashed=False):
         """
         Draw a line on the given Cairo context `cr` from (`x1`, `y1`) to
         (`x2`,`y2`), and update the bounding rectangle if necessary.
+
+        For a dashed line, `dashed` be a list of on/off segment lengths,
+        or `True` in which case [3.5,3.5] is used.
         """
 
         cr.set_source_rgba(0.0, 0.0, 0.0, 1.0)
         cr.set_line_width(1.0)
         if dashed:
-            cr.set_dash([3.5])
+            if dashed is True:
+                dashed = [3.5]
```

Comment:
The docsting says `[3.5, 3.5]`, here it's `[3.5]`, which is correct?
Could you also provide a general explanation for why `dash` is sometimes a `bool` and sometimes a `list`? I understand it wasn't introduced by this PR.

Reply:
Other places in the code were already calling this method with `dashed=True` and get a default dash size of 3.5. I wanted a way to specify the dash size.  I guess it could be done with an extra optional argument `dash_size` instead of overloading this one if you think that's better.

(`[3.5]` is equivalent to `[3.5, 3.5]` when specifying dash sizes in cairo, but to avoid people needing to know that I've changed it to `[3.5, 3.5]` in a fixup commit)

Reply:
If this arg is currently being used only as a `bool`, let's make a new one, unless you think it's very redundant

Reply:
This better? https://github.com/ReactionMechanismGenerator/RMG-Py/pull/1961/commits/c695be3f596d2f81960b379bfb1baeb29d499984

Reply:
yes, thanks!